lightwood/mixer/sktime.py

```diff
@@ -291,15 +291,30 @@ def _call_groupmodel(self,
         if hasattr(submodel, '_cutoff') and hasattr(submodel, 'd'):
             model_d = 0 if submodel.d is None else submodel.d
             cutoff = submodel._cutoff.values[0] if isinstance(submodel._cutoff, pd.Int64Index) else submodel._cutoff
-            min_offset = -cutoff + model_d + 1
+            # min_offset = -cutoff + model_d + 1
+            # this will fix the wrong alignment mention in
+            # https://github.com/mindsdb/mindsdb/issues/3213
+            min_offset = -cutoff + model_d
         else:
             min_offset = -np.inf
 
         start = max(offset, min_offset)
         end = series.shape[0] + offset + self.horizon
-        all_preds = model.predict(np.arange(start, end)).tolist()
+        # all_preds = model.predict(np.arange(start, end)).tolist()
+        # StatsForcastAutoARIMA return wrong prediction with data point leq zero
+        # Here a WA proposed by getting prediction for ALL trained data and
+        # slice it again from the last data point
+        if start == 0:
+            start = min_offset
+            all_preds = model.predict(np.arange(start, end)).tolist()[-min_offset:]
```

Comment:
Yeah, this workaround will do for now. It may be slightly more compute, but we can remove it once the behavior is fixed in the dependency.